pkg/core/codeset_store.go

```diff
@@ -37,6 +38,16 @@ func (cs *GitCodesetStore) Find(ctx context.Context, project, name string) (*dom
 	return result, nil
 }
 
+// Delete removes a codeset identified by project and name
+func (cs *GitCodesetStore) Delete(ctx context.Context, project, name string) error {
+	err := cs.gitAdmin.DeleteRepository(project, name)
+	// TODO should we delete the project+user too? If it does not contain any repos?
```

Comment:
Better to keep the project+user for now, because we'll need to model those as independent concepts anyway at some point.